jax/experimental/mosaic/gpu/profiler.py

```diff
@@ -69,7 +69,7 @@ def _event_elapsed(start_event, end_event):
   )(start_event, end_event)
 
 
-def measure(
+def _measure_events(
     f: Callable[P, T], *args: P.args, **kwargs: P.kwargs
 ) -> tuple[T, float]:
   """Measures the time it takes to execute the function on the GPU.
```

Comment:
This docstring should be move to the public `measure` function and extended to explain `mode` and `aggregate`.

Reply:
I added a detailed docstring to `measure`.